### ISSUE TITLE
Warning messages Run 1

### ISSUE THREAD

**Issue by aubombarely** [LongTianPy/WGD#2]
I have these warning messages:
1- *\* (WGD_detection.py:10372): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Cf5K2FmXPG: Connection refused
Translating CDS to proteins...
2- usr/local/lib/python2.7/dist-packages/Bio/Seq.py:2041: BiopythonWarning: Partial codon, len(sequence) not a multiple of three. Explicitly trim the sequence or add trailing N before translation. This may become an error in future.
  BiopythonWarning)
